packages/devextreme/js/ui/text_box/ui.text_editor.label.js

```diff
@@ -139,9 +139,11 @@ class TextEditorLabel {
     }
 
     _updateBeforeWidth() {
-        this._$before.css({ width: this._props.beforeWidth });
+        if(this._isVisible()) {
+            this._$before.css({ width: this._props.beforeWidth ?? this._props.getBeforeWidth() });
```

Comment:
let's move it to a variable

Reply:
https://github.com/DevExpress/DevExtreme/pull/26247/commits/5322af04efe33ef817500989e942071703dbcb28